lib/wasm/src/sections_translator.rs

```diff
@@ -17,6 +17,10 @@ use wasmparser;
 use std::str::from_utf8;
 use environ::ModuleEnvironment;
 
+// this is for no_std builds, but has no affect on regular builds
```

Comment:
Actually, I just noticed that these comments appear in some places that do this and not others, and it seems like it'd be good to be consistent about it. My instinct is that we could probably just drop these comments, but I don't have a strong feeling either way.

Reply:
The majority of the work I did on porting was a couple of weeks ago. My styles appear to have changes since then, I'll make it consistent.